crates/subspace-farmer-components/src/sector.rs

```diff
@@ -503,8 +503,7 @@ impl SectorContentsMap {
     /// corresponds to the record to which chunk belongs and `encoded_chunk_used` indicates whether
     /// it was encoded.
     ///
-    /// ## Panics
-    /// Panics if `s_bucket` is outside of [`Record::NUM_S_BUCKETS`] range.
+    /// Returns error if `s_bucket` is outside of [`Record::NUM_S_BUCKETS`] range.
```

Comment:
Thanks for fixing this message, I recall it was panicking at some point in the past, but not anymore